akka-docs/src/main/paradox/coexisting.md

```diff
@@ -0,0 +1,93 @@
+# Coexistence
+
+We believe Akka Typed will be adopted in existing systems gradually and therefore it's important to be able to use typed 
+and untyped actors together, within the same `ActorSystem`. Also, we will not be able to integrate with all existing modules in one big bang release and that is another reason for why these two ways of writing actors must be able to coexist.
+
+There are two different `ActorSystem`s: `akka.actor.ActorSystem` and `akka.actor.typed.ActorSystem`. 
+The latter should only be used for greenfield projects that are only using typed actors. The `akka.actor.ActorSystem`
+can be adapted so that it can create typed actors.
+
+Typed and untyped can interact the following ways:
+
+* untyped actor systems can create typed actors
+* typed actors can send messages to untyped actors, and opposite
+* spawn and supervise typed child from untyped parent, and opposite
+* watch typed from untyped, and opposite
+
+In the Scala examples the `akka.actor` package is aliased to `untyped`. The Java examples use fully quaified
+class names for the untyped classes.
+
+Scala
+:  @@snip [UntypedWatchingTypedSpec.scala]($akka$/akka-actor-typed-tests/src/test/scala/docs/akka/typed/coexistence/UntypedWatchingTypedSpec.scala) { #import-alias }
+
+### Untyped to typed 
+
+For the following typed actor here's how you create, watch and send messages to
+it from an untyped actor.
+
+Scala
+:  @@snip [UntypedWatchingTypedSpec.scala]($akka$/akka-actor-typed-tests/src/test/scala/docs/akka/typed/coexistence/UntypedWatchingTypedSpec.scala) { #typed }
+
+Java
+:  @@snip [UntypedWatchingTypedTest.java]($akka$/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/coexistence/UntypedWatchingTypedTest.java) { #typed }
+
+The top level untyped actor is created from an untyped actor system:
+
+Scala
+:  @@snip [UntypedWatchingTypedSpec.scala]($akka$/akka-actor-typed-tests/src/test/scala/docs/akka/typed/coexistence/UntypedWatchingTypedSpec.scala) { #create-untyped }
+
+Java
+:  @@snip [UntypedWatchingTypedTest.java]($akka$/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/coexistence/UntypedWatchingTypedTest.java) { #create-untyped }
+
+Then it can create a typed actor, watch it, and send a message to it:
+
+Scala
+:  @@snip [UntypedWatchingTypedSpec.scala]($akka$/akka-actor-typed-tests/src/test/scala/docs/akka/typed/coexistence/UntypedWatchingTypedSpec.scala) { #untyped-watch }
+
+Java
+:  @@snip [UntypedWatchingTypedTest.java]($akka$/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/coexistence/UntypedWatchingTypedTest.java) { #untyped-watch }
+
+There is one `import` that is needed to make that work for Scala. For Java we import the Adapter class and
```

Comment:
same here, adjust the text with the `@java` and `@scala` thing

Reply:
Done